omniduct/databases/druid.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+
+from omniduct.utils.debug import logger
+
+from .base import DatabaseClient
+
+
+class DruidClient(DatabaseClient):
+
+    PROTOCOLS = ['druid']
+    DEFAULT_PORT = 80
+
+    def _init(self):
+        self.__druid = None
+
+    # Connection
+    def _connect(self):
+        from pydruid.db import connect
+        logger.info('Connecting to Druid database ...')
+        self.__druid = connect(self.host, self.port, path='/druid/v2/sql/', scheme='http')
+
+    def _is_connected(self):
+        return hasattr(self, '__druid') and self.__druid is not None
+
+    def _disconnect(self):
+        logger.info('Disconnecting from Druid database ...')
+        try:
+            self.__druid.close()
+        except Exception:
```

Comment:
Do you know what the Druid client raises if it is not connected?

Reply:
It's sometimes buggy so I'm going to leave this as is for now
https://github.com/druid-io/pydruid/pull/119